app/models/user.rb

```diff
@@ -20,6 +20,28 @@ def self.digest string
     BCrypt::Password.create string, cost: cost
   end
 
+  class << self
+    def new_token
+      SecureRandom.urlsafe_base64
+    end
+  end
+
+  attr_accessor :remember_token
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
```

Comment:
```suggestion
    update_column :remember_digest, User.digest(remember_token)
```
với trường hợp này, ri mới hợp lý nhé